src/cmd/shared.go

```diff
@@ -38,6 +41,9 @@ func getSecrets(awsConfig types.AwsConfig) types.Secrets {
 }
 
 func getBaseDeployConfig(appContext *context.AppContext, remoteEnvironmentID string) deploy.Config {
+	if awsProfileFlag == "" {
+		awsProfileFlag = appContext.Config.Name
+	}
```

Comment:
I don't think this is needed since it calls `getAwsConfig` which defaults it to the app name